app/api/tests/data/file.fast

```diff
@@ -0,0 +1,10 @@
+__label__sauce __label__cheese How much does potato starch affect a cheese sauce recipe?
```

Comment:
rename file. `.fast` is not the preferred extension. e.g. `example_fasttext.txt`